server/app.js

```diff
@@ -32,6 +32,18 @@ const csrfProtection = csrf({ cookie: true })
 const parseForm = bodyParser.urlencoded({ limit: '50mb', extended: true })
 const app = express();
 
+if(process.env.NODE_ENV === 'development') {
+  const livereload = require('livereload')
+  const connectLiveReload = require('connect-livereload')
+  const liveReloadServer = livereload.createServer();
+  liveReloadServer.watch(path.join(__dirname, '../public'));
+  liveReloadServer.server.once("connection", () => {
+    setTimeout(() => {
+      liveReloadServer.refresh("/");
```

Comment:
does this mean that it will refresh the app to the `"/"` route on changes? What if we're on a different route, will we be sent back to the homepage?

Reply:
Good question. The live reload will refresh it's current page, there would be no rerouting.
This is a [quick video](https://drive.google.com/file/d/1HhDshYzqFL3ljivYviS3Wsat5P5UySRw/view?usp=sharing) of it in action. 
In the video I am editing the login page, then navigating to sign up and editing sign up.